react/features/base/tracks/actions.native.ts

```diff
@@ -37,12 +37,11 @@ export function toggleScreensharing(enabled: boolean, _ignore1?: boolean, _ignor
                 _startScreenSharing(dispatch, state);
```

Comment:
I think we should start the service in the `.then()` so after `_startScreenSharing()` resolves.